browser/bootstrap.js

```diff
@@ -27,6 +28,32 @@ const inspectElementItem = new MenuItem({
   }
 });
 
+function openAboutWindow() {
+
+  let aboutWindow = new BrowserWindow({
```

Comment:
Should be:
``` JavaScript
let aboutWindow = new BrowserWindow({
    parent: remote.getCurrentWindow(),
    modal: true,
    useContentSize: true,
    width: 565,
    height: 355,
    'autoHideMenuBar': true,
    resizable: false,
    show: false
  });
```
to keep original background size.

That would get us even closer to original design, but still vertical spacing between text requires some adjustments.

![image](https://user-images.githubusercontent.com/620330/30470319-17ec14e8-99a9-11e7-826e-7a55aa3892f3.png)